src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+
+    public static String generateRandomColor() {
```

Comment:
[Don't use static methods in your solution](https://github.com/mate-academy/jv-oop-advanced/blob/master/checklist.md#dont-use-static-methods-in-your-solution)